WasteManagement/ngsild-payloads/Site.jsonld

```diff
@@ -0,0 +1,56 @@
+{
+	"id":"urn:ngsi-ld:Site:01",
+	"type":"Site",
+	"location":{
+		"type":"GeoProperty",
+		"value":{
+			"type":"Point",
+			"coordinates":[
+				-8.5,
+				41.2
+			]
+		}
+	},
+	"name":{
+		"type":"Property",
+		"value":"Sophia"
+	},
+	"illegalDeposit":{
```

Comment:
I wonder if illegaldeposit should not be an entity (observedIn a Site)

Reply:
for top level directory (and for the generic URI also), I think that giles may help us, because the smartCamera now can detect other thins like number of person in the site and other functionalities that we will add..

Reply:
illegaldeposit  is detected by the sensor of the smartCamera.. So as I did It for all models, the value have to be attached to the concerned Entity (eg temperature in the FishContainement)

Reply:
does that mean we only want to keep the last illegat deposit on a site (at least in the graph) ?

we may be able to handle it as any temporal property but not sure it fits well in such a model.